packages/itwinui-css/scripts/createComponent.js

```diff
@@ -50,14 +50,6 @@ const componentIndexFactory = (directory) => {
   };
 };
 
-/** Appends component index to global index.scss */
-const globalIndexFactory = (directory) => {
```

Comment:
We don't have global index anymore, right?

Reply:
Right